lib/mutant/mutator/node/arguments.rb

```diff
@@ -79,12 +74,11 @@ def emit_mlhs_expansion
           end
         end
 
-        # Return mlhs childs
+        # The mlhs childs
```

Comment:
Multiple  left hand side. For people dealing with the ruby AST its more common https://github.com/whitequark/parser/blob/master/doc/AST_FORMAT.md#multiple-left-hand-side. I think expanding it makes sense.